app/mailers/feedback_mailer.rb

```diff
@@ -13,6 +13,6 @@ def feedback(sending_user, form_values)
   end
 
   def self.support_email
-    ENV['SUPPORT_EMAIL'] || 'gatewaycommunicator@gsa.gov' # not sensitive, so hard coding
+    ENV['SUPPORT_EMAIL'] || 'tickets@18f.uservoice.com' # not sensitive, so hard coding
```

Comment:
something like support@c2.18f.gov ? That particular domain is pointed at mandrill which does not seem able to simply redirect mail to a different destination. We can continue to use gatewaycommunicator@gsa.gov as our published address and forward via gmail afaik.